src/coreclr/vm/jithelpers.cpp

```diff
@@ -572,6 +572,38 @@ FORCEINLINE INT64 FastDbl2Lng(double val)
 #endif
 }
 
+//------------------------------------------------------------------------
+// TruncateDouble: helper function to truncate double 
+//                 numbers to nearest integer (round towards zero).
+//
+// Arguments:
+//    val  - double number to be truncated.
+//
+// Return Value:
+//    double: truncated number (rounded towards zero)
+// 
+double TruncateDouble(double val)
+{
+    FCALL_CONTRACT;
```

Comment:
```suggestion
    LIMITED_METHOD_CONTRACT;
```
FCALL_CONTRACT should be only used on FCalls itself

Reply:
Thanks, I have made the suggested change.